testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt

```diff
@@ -0,0 +1,38 @@
+package com.rakuten.tech.mobile.testapp.ui.input
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.textfield.TextInputEditText
+import com.rakuten.tech.mobile.miniapp.testapp.R
+import com.rakuten.tech.mobile.testapp.ui.display.MiniAppDisplayActivity
+import kotlinx.android.synthetic.main.mini_app_input_activity.*
+
+class MiniAppInputActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.mini_app_input_activity)
+
+        btnDisplay.setOnClickListener(onDisplayClick)
+    }
+
+    private val onDisplayClick = View.OnClickListener {
+        val isAppIdValid = isValidInput(edtAppId)
+        val isVersionIdValid = isValidInput(edtVersionId)
+
+        if (isAppIdValid && isVersionIdValid)
+            MiniAppDisplayActivity.start(this,
+                edtAppId.text.toString().trim(), edtVersionId.text.toString().trim())
```

Comment:
We were talking about changing the public interface to only accept `MiniAppInfo` (`MiniApp#create(info: MiniAppInfo`) and adding `MiniApp#getInfo(miniAppId: String): MiniAppInfo` for creating a Mini App when given only an App Id.

So I'm not sure how this fits into that. I guess we can keep this for now and modify it later when we change the public interface.